cookiecutter-django-app/{{cookiecutter.repo_name}}/.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Python CI
```

Comment:
This file is also a duplicate

Reply:
This should also be layered in from the python-template

Reply:
workflow setup in django-app is setting up specific Django versions for CI that python-library doesn't need

Reply:
ah okay, sorry, I should have looked more carefully.